src/main/java/com/puppycrawl/tools/checkstyle/meta/XmlMetaReader.java

```diff
@@ -40,6 +40,7 @@
 
 /**
  * Class having utilities required to read module details from an XML metadata file of a module.
+ * Note: methods of this class are used in Eclipse plugin.
```

Comment:
This class is used by plugins that need load of metadata from XML files.

As it is for all who need  tool to load our files

Reply:
done

P.S. I also refactored XmlMetaReader#read method more and added test.